packages/app-desktop/index.html

```diff
@@ -31,6 +31,9 @@
 				background: #CF3F00;
 				color: white;
 			}
+			.CodeMirror-line  {
+				unicode-bidi: plaintext;
```

Comment:
Could you add a comment here to explain what this change does, along with a link to the issue? https://github.com/laurent22/joplin/issues/3991

Reply:
Added.